### PR TITLE
antisnake

### PR DESCRIPTION
**Pull request by plankton9991**
- [DEMO LINK](https://plankton9991.github.io/layout_antisnake/)
- [TEST REPORT LINK](https://plankton9991.github.io/layout_antisnake/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/box.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  grid-template-columns: 1fr 1fr 1fr;
+
+  @media (max-width: 899px) {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  @media (max-width: 599px) {
+    grid-template-columns: 1fr;
+  }
+
```
Comment (Jenia80) [inline src/styles/blocks/box.scss:12]: i think it possible to use only one media and border box

File: src/styles/blocks/box.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    &:nth-child(5) {
+      background: rgb(51, 0, 0);
+    }
+
+    &:nth-child(6) {
+      background: rgb(0, 0, 0);
+    }
+  }
+}
```
Comment (Jenia80) [inline src/styles/blocks/box.scss:42]: try to use variables for color

**Review by Jenia80 — APPROVED** [review]

File: src/styles/blocks/box.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.box {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
```
Comment (AlinaMatuschak) [inline src/styles/blocks/box.scss:3]: It will be great to create a variable for the box size

**Review by AlinaMatuschak — APPROVED** [review]
Good work 👌

**Pull request closed without merging**